utils/rpms/daos.spec

```diff
@@ -346,11 +347,18 @@ getent passwd daos_server >/dev/null || useradd -s /sbin/nologin -r -g daos_serv
 %post server
 /sbin/ldconfig
 %systemd_post %{server_svc_name}
+set -o noclobber
+cat << EOF > %{sysctl_script_name}
+net.core.rmem_max=104857 +
+net.core.rmem_default=104857
```

Comment:
Would be great to verify whether the existing values are set to a larger value already.

Reply:
I think doing so would turn this into a run-time check/set.  But I do understand @johannlombardi's concern here.

Or, maybe rather than being `99-daos_server.conf` it should be more along the lines of `10-daos_server.conf`, giving the admin plenty of opportunity to add his own override to that in a file named > 10.

Reply:
what changes should I make to satisfy the requests? should I still create a config file but rename to 10-*.conf? should I add a sysctl rmem value check on line 350 of daos.spec before applying the config? @brianjmurrell @johannlombardi

Reply:
My personal feeling is that it should be installed as a file, labeled as a configuration file, at a low priority like 10-ish.  Then the sysadmin can override this with his own value at a higher priority if he wishes.

If one really wanted to, I am sure one could come up with some `%post` code to hunt down any other `/etc/sysctl.d/` files setting the same parameters as we are setting and at least emit a warning about the potential conflict, but that would be totally optional gravy IMHO.

If/when a sysadmin notices that his custom settings are not being honoured, it should not take him too long at all to discover why, and I would even say the potential is low for this if we use a lower priority, like 10-ish.

Probably worth adding that we are setting these, and how, in a _Release Note_.